src/Microsoft.Azure.WebJobs.Logging.ApplicationInsights/ApplicationInsightsLogger.cs

```diff
@@ -397,6 +402,11 @@ private static void ApplyFunctionResultActivityTags(IEnumerable<KeyValuePair<str
                 {
                     currentActivity.AddTag(LogConstants.LogLevelKey, logLevel.ToString());
                 }
+
+                if (scope.TryGetValue(ApplicationInsightsScopeKeys.HttpRequest, out var request) && request is HttpRequest httpRequest)
+                {
+                    currentActivity.AddTag(LoggingConstants.ClientIpKey, GetIpAddress(httpRequest));
```

Comment:
Down in `GetIpAddress()` (I can't add a comment there b/c it's not part of your change) -- we need to change it to be...

` return httpRequest.HttpContext?.Connection?.RemoteIpAddress?.ToString() ?? LoggingConstants.ZeroIpAddress;`

...to allow for null `Connection` or `RemoteIpAddress`. The tests in Script that are using TestServer don't have values here so I'm getting a null ref. I'd only been testing this change e2e so hadn't bumped into this yet.